doc/use-cases.rst

```diff
@@ -249,6 +270,153 @@ management platforms, forensic frameworks...) using Yeti's API::
+                        "matched_observable": "C:\\Users\\admin\\AppData\\Roaming\\Ijhsz",
+                        "name": "Dridex stolen data",
+                        "pattern": "[A-Z]:\\\\Users\\\\[a-z]+\\\\AppData\\\\Roaming\\\\[A-Z][a-z]{3}",
+                        "type": "Regex",
+                        "url": "http://localhost:5000/api/indicator/5a1ae41610c5537472a8cbd5"
+                    }
+                ]
+            },
+            "name": "Dridex",
+            "tags": [
+                "dridex"
+            ],
+            "type": "Malware",
+            "url": "http://localhost:5000/api/entity/5a1add5c10c5537472a8cbd1"
+        }
+    ],
+    "known": [],
+    "matches": [
+        {
+            "description": "",
+            "diamond": "target",
+            "human_url": "http://localhost:5000/indicator/5a1ae41610c5537472a8cbd5",
+            "id": "5a1ae41610c5537472a8cbd5",
+            "location": "Filesystem",
+            "name": "Dridex stolen data",
+            "observable": "C:\\Users\\admin\\AppData\\Roaming\\Ijhsz",
+            "pattern": "[A-Z]:\\\\Users\\\\[a-z]+\\\\AppData\\\\Roaming\\\\[A-Z][a-z]{3}",
+            "related": [
+                {
+                    "entity": "Malware",
+                    "link_description": "Indicates",
+                    "name": "Dridex"
+                }
+            ],
+            "suggested_tags": [
+                "dridex",
+                "banker"
+            ],
+            "type": "Regex",
+            "url": "http://localhost:5000/api/indicator/5a1ae41610c5537472a8cbd5"
+        }
+    ],
+    "neighbors": [],
+    "unknown": [
+        "C:\\Users\\admin\\AppData\\Roaming\\Ijhsz"
+    ]
+  }
+
+
+Ingesting and enriching a third-party report
+--------------------------------------------
+
+An abundant source (but of uneven quality) of intelligence on adversaries can be found
+in the different reports that are published by vendors, CERTs, etc. Yeti provides
+an easy way to parse information from a variety of sources (blogposts, PDF files, raw text)
+and integrate them in Yeti.
+
+Starting an Investigation
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The "ingestion" of a report in Yeti parlance typically means the creation of an
+Investigation, from which you will be able to add observables and tie them to existing or new Entities
+like Actors, Malware, or TTPs.
+
+Let's try creating an investigation out of this blogpost. Head to **Investigations > Import**::
+
+  https://badcyber.com/technical-analysis-of-recent-attacks-against-polish-banks/
+
+.. image:: _static/investigation_new.png
+
+.. note:: Investigations can be started on your own, from a single Observable or Entity,
+  using the ``New Investigation`` button on their corresponding page.
+
+Clicking on **Start Import** will lead you to the Import page where all the magic happens.
+On the **left pane** is the parsed Markdown document corresponding to the document you just imported.
```

Comment:
It should only show the markdown document if the installation does not have "wkhtmltopdf" or if it failed. Otherwise, it should show a PDF of the blog post.

Reply:
Fixed